### DISCUSSION TITLE
希望能出返回哔哩哔哩旧版的功能

### DISCUSSION THREAD

**Discussion by xiaolong22333** [the1812/Bilibili-Evolved#1410] [功能建议 / Ideas]
原来哔哩哔哩一直用的是旧版，用了这个脚本后自动换成新版了，很不习惯，而且哔哩哔哩现在不能让用户选择回到旧版。。。
希望万能的作者可以实现这个小小的功能^_^

**Comment by Geeyun-JY3** [comment]
换成新版和脚本没关系，19年到20年各个板块都有大大小小的改版。
建议出门左转：https://github.com/MotooriKashin/Bilibili-Old

还有这功能可一点都不小。
